Dockerfile

```diff
@@ -0,0 +1,13 @@
+#Creates a Container with everything needed to generate the EFI image
+#On run it generates the image from sources in /zbm to /zbm out. Mount at your will
+#see docker-compose.yml
+FROM bougyman/voidlinux:latest
+WORKDIR /zbm
+RUN xbps-install -SAy make dracut perl gcc linux gummiboot-efistub zfs kexec-tools ncurses fzf skim mbuffer
+#the fisrt cpan install fails because I have no idea how to use it correctly
+#in a dockerfile, but the following ones do work. Somebody please fix.
+RUN cpan install Sort::Versions || cpan install Sort::Versions Config::IniFiles YAML::PP boolean
+ADD . /zbm
```

Comment:
As already said I would not git pull it again because I suppose the workflow is more like check out the source, maybe modify it and then use docker to build it, so the source is already checked out and should be used. However I think the during build you can only use the contents of the current directory, so the Dockerfile must be in the root directory or we need the context directive in docker-compose https://stackoverflow.com/questions/24537340/docker-adding-a-file-from-a-parent-directory